back-nest/src/redis/redis.service.ts

```diff
@@ -47,6 +49,11 @@ export class RedisService implements OnModuleInit, OnModuleDestroy {
     this.logger.log('Cache in current Redis has been cleared');
   }
 
+  async deleteProductCache(): Promise<void> {
+    await this.redisClient.del(CacheKeys.PRODUCTS);
+    this.logger.log('Cache for the key ${CacheKeys.PRODUCTS} has been cleared');
```

Comment:
Re-check this twice :). It will now work in the way you've made it.

[How to use template string](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/raw)